Code/GraphMol/RGroupDecomposition/RGroupData.h

```diff
@@ -86,7 +96,7 @@ struct RGroupData {
 
  private:
   std::string getSmiles()
-      const {  // compute the canonical smiles for the attachments
+      const {  // compute the canonical smiles for the attachments (bug: removes dupes since we are using a set...)
```

Comment:
is it really a bug?

Reply:
I think it might be, It will make equivalent:
```
[*:1](O)O
[*:1](O)(O)O
```

But I might have convinced myself that it is a feature.